kratos/python_scripts/read_materials_process.py

```diff
@@ -166,9 +166,12 @@ def _AssignPropertyBlock(self, data):
         mat = data["Material"]
 
         # Set the CONSTITUTIVE_LAW for the current properties.
-        constitutive_law = self._GetConstitutiveLaw( mat["constitutive_law"]["name"].GetString() )
+        if (mat.Has("constitutive_law")):
```

Comment:
do you need to modify this file? i think there is already a PR with this proposed modification. We'll consider this change in that PR

Reply:
This is already merge in the master

https://github.com/KratosMultiphysics/Kratos/pull/2217

Reply:
ok i am accepting this